06_modules_lab/01.py

```diff
@@ -1,4 +1,5 @@
-""" Write a program that takes a count
-from sys.argv import and prints "Hello Python"
-count times
-"""
+from sys import argv
+
+(script_name, lines) = argv

```

Comment:
Since nobody uses `script_name` it's better to name it `_` (this is python's convention for a variable nobody cares about)